<?xml version="1.0"?>
<launch>

  <!--- <param name="navfn/allow_unknown" value="false"/>-->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find landshark_2dnav)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find landshark_2dnav)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find landshark_2dnav)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find landshark_2dnav)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find landshark_2dnav)/param/dwa_local_planner_params.yaml" command="load" /> 
    <rosparam file="$(find landshark_2dnav)/param/navfn_global_planner_params.yaml" command="load" /> 
    <rosparam file="$(find landshark_2dnav)/param/move_base_params.yaml" command="load" /> 

    <remap from="odom" to="landshark/odom"/>    
<!--    <remap from="odom" to="landshark/odom_stub"/>    -->

  </node>
</launch>
